src/store/validation/validationHelpers.ts

```diff
@@ -121,8 +121,15 @@ export function validateCommunityBlock({block, schemaVersion}: {block: IBlock, s
   let validate = null
   if (isEmpty(validators) || !validators.has(block.type)) {
     const blockTypeWithoutNameSpace = block.type.split('.')[block.type.split('.').length - 1]
-    const blockJsonSchema = require(`@floip/flow-runner/dist/resources/validationSchema/${schemaVersion}/I${blockTypeWithoutNameSpace}Block.json`)
-    validate = createDefaultJsonSchemaValidatorFactoryFor(blockJsonSchema)
+    let blockJsonSchema
+    try {
+      blockJsonSchema = require(`@floip/flow-runner/dist/resources/validationSchema/${schemaVersion}/I${blockTypeWithoutNameSpace}Block.json`)
+      validate = createDefaultJsonSchemaValidatorFactoryFor(blockJsonSchema)
+    } catch (e) {
+      console.info(`A Specific Validator for the ${blockTypeWithoutNameSpace}Block could not be found. Falling back the generic Block validator for ${schemaVersion}`)
```

Comment:
Perhaps this could be a warning but I'm not sure we want to log the whole error message as this is a supported state (when you have a new block but haven't set up a validator yet)